next-desolates-map/components/SettingsPage/SettingsPage.tsx

```diff
@@ -0,0 +1,466 @@
+
+                        {/* Planet Input */}
+                        <li className="">
+                            <div className="min-h-0 lg:ml-3 h-8 w-full bg-black bg-opacity-75 rounded-lg outline-cool mb-5 lg:mb-1">
+                                    <input
+                                        id="search-input"
+                                        type="text"
+                                        name="planet-no"
+                                        size={40}
+                                        className="w-full py-1 pl-4 text-lg text-left bg-transparent rounded-r-lg text-primary font-cool placeholder-primary focus:outline-none"
+                                        placeholder="Enter planet name..."
+                                        onChange={handleInputChange}
+                                        autoComplete="off"
+                                        onKeyDown={handleKeyDown}
+                                    >
+                                    </input>
+                                </div>
+                        </li>                        
+
+                        </ul>
+
+                    {/* Cancel & Save btns */}
+
+                    <div className="min-h-0 max-h-screen  h-8 flex justify-end">
+                        <button
+                            className={`font-cool p-2 mr-3 text-sm text-center text-primary transition-colors duration-200 rounded-lg hover:text-white hover:bg-primary w-20 outline-cool backdrop-filter backdrop-blur bg-faded`}
+                        >
+                            CANCEL
+                        </button>
+
+                        <button
+                            className={`font-cool p-2 text-sm text-center text-primary transition-colors duration-200 rounded-lg hover:text-white hover:bg-primary w-20 outline-cool backdrop-filter backdrop-blur bg-faded`}
+                        >
+                            SAVE
+                        </button>
+                    </div>
+
+                </div>
+
+            </div>
+        </div>   
+        </div>
+    );
+}
+
+interface IClusterButton {
+    cluster: Clusters;
+}
+
+function ClusterButton({ cluster }: IClusterButton) {
+    const { changeStage, setOngoingCamMov, p5, cam, changeCurrentCluster } =
+        useContext(SpaceRendererContext);
+
+    function handleClick(e: any) {
+        changeCurrentCluster(cluster);
+
+        let tempCamMov = new CameraMovement(
+            p5!,
+            cam!,
+            p5!.createVector(0, 0, 0),
+            2000,
+            1000
+        );
+
+        tempCamMov.start(p5!);
+
+        setOngoingCamMov(tempCamMov);
+
+        changeStage(Stages.SPACE_NAVIGATION);
+    }
+
+    return (
+        <button
+            onClick={handleClick}
+            className="w-full p-4 text-left transition-colors duration-200 rounded-lg outline-cool backdrop-filter backdrop-blur bg-faded hover:text-white hover:bg-primary"
```

Comment:
We don't seem to need all of this. I would remove it

Reply:
Done